install

```diff
@@ -384,7 +384,7 @@ Dir.chdir HOMEBREW_REPOSITORY do
     curl_flags += "k" if mac? && macos_version < "10.6"
     core_tap = "#{HOMEBREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
     system "/bin/bash -o pipefail -c 'curl -#{curl_flags} #{BREW_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
-
+    
```

Comment:
Almost there, @Bonsitar ! You've added some additional white space here at line 387. Please remove them.